### DISCUSSION TITLE
feat: add a "fortran.formatting.fprettify.configure" option

### DISCUSSION THREAD

**Discussion by pums974** [krvajal/vscode-fortran-support#705] [Q&A - deprecated see Fortran Discourse]
### Is there an existing request for this?

- [X] I have searched the existing issues

### Feature Request

It would be similar to  `fortran.fortls.configure` and can replace 

```json
"fortran.formatting.fprettifyArgs": [
         "-c",
         ".fprettify.rc"
    ],
```
which doesn't work (I suspect that it doesn't find the file in question, which is at the root of the `workspaceFolder`).

**Comment by gnikit** [comment]
There is this option available 

https://github.com/fortran-lang/vscode-fortran-support/blob/1f2fe6d8465a80109501ee22eabbcf4347eb7d7d/package.json#L363

See "Feature Contributions" in VS Code for all the available settings

![image](https://user-images.githubusercontent.com/16143716/194855855-836cd0bf-0141-489f-bbc7-023b024671b5.png)

**Comment by pums974** [comment]
I know that but, as I said, it doesn't work here for some reason.
As soon I add this option, the formatter stop working, and I have nothing in the logs.
I can now confirm that this is due to relative path. Using absolute path works.
